### ISSUE TITLE
Remove `elm-bulma-classes` dependency

### ISSUE THREAD

**Issue by surprisetalk** [surprisetalk/elm-bulma#9]
Unfortunately, `elm-bulma-classes` is no longer being maintained.
I'm thinking that we should make an equivalent, either in this package or a separate one.
We're open to suggestions!

**Comment by ahstro** [comment]
FYI, I made [something similar](https://github.com/ahstro/elm-bulma-classes) for my own use, you might want to consider using it here, unless you need something more complex :) I'd also definitely consider giving you write access so you can update it yourself if you need an update quickly.

**Comment by surprisetalk** [comment]
Thanks @ahstro !
Your library is super clean and helpful :smiley_cat: I'm very tempted to add it as a dependency for the next release.

Just curious, why did you decide to export the classes as `String`s, rather than `Html.Attribute`s?

**Comment by ahstro** [comment]
Hmm, didn't even consider it. Just off the top of my head, it's easier to conditionally use classes (with `Html.Attributes.classList`) if they're `String`s. I'll think some more about it; feel free to open up an issue if you want to weigh in :)

**Comment by woylie** [comment]
Apparently, the elm-bulma-classes repository has been removed from Github, so it's not even possible to install elm-bulma right now.

```
elm-package install surprisetalk/elm-bulma
Some new packages are needed. Here is the upgrade plan.

  Install:
    danielnarey/elm-bulma-classes 2.0.15
    surprisetalk/elm-bulma 6.0.2

Do you approve of this plan? [Y/n] y
Starting downloads...

  ● surprisetalk/elm-bulma 6.0.2
  ✗ danielnarey/elm-bulma-classes 2.0.15
Error: The following HTTP request failed.
<https://github.com/danielnarey/elm-bulma-classes/zipball/2.0.15/>

<!DOCTYPE html>
<html>
  <head>
    <meta http-equiv="Content-type" content="text/html; charset=utf-8">
    <meta http-equiv="Content-Security-Policy" content="default-src 'none'; base-uri 'self'; connect-src 'self'; form-action 'self'; img-src data:; script-src 'self'; style-src 'unsafe-inline'">
    <meta content="origin" name="referrer">
    <title>Page not found &middot; GitHub</title>
(...)
```

**Comment by surprisetalk** [comment]
Thanks for the heads-up @woylie ! 

I just released version 6.1.0 without the `elm-bulma-classes` dependency. Let me know if it works for you!
